annotations-runtime/src/main/java/com/github/dynamicextensionsalfresco/schedule/ScheduledTaskRegistrar.java

```diff
@@ -42,7 +42,9 @@ public void afterPropertiesSet() {
     }
 
     public void registerScheduledTaskAnnotatedBeans() {
-        Map<String, Object> scheduledBeans = applicationContext.getBeansWithAnnotation(ScheduledQuartzJob.class);
+        Map<String, Object> scheduledBeans = applicationContext.getBeansWithAnnotation(ScheduledTask.class);
+        scheduledBeans.putAll(applicationContext.getBeansWithAnnotation(ScheduledQuartzJob.class));
```

Comment:
Whoops, how did I miss that ?
And how did you catch that ?

Reply:
Was updating the example-dynamic-extensions project to make it use the `@ScheduledTask` annotation & I noticed it didn't work